src/AppKit/ActionDispatcher.cs

```diff
@@ -40,25 +42,25 @@ internal class ActionDispatcher : NSObject
 		const string dkey = "__monomac_internal_ActionDispatcher_doubleActivated:";
 		public static Selector Action = new Selector (skey);
 		public static Selector DoubleAction = new Selector (dkey);
-		public EventHandler Activated;
-		public EventHandler DoubleActivated;
+		public EventHandler? Activated;
```

Comment:
In previous places I've worked, we liked to set up `EventHandler` members like this:
```
public EventHandler Activated = (s, e) => { };
```
This lets you turn the `OnActivated` into a one-liner:
```
[Preserve, Export (skey)]
public void OnActivated (NSObject sender) => handler (sender, EventArgs.Empty);
```

Reply:
That uses more memory, because you'd always create an `EventHandler` instance, even if nobody adds an event handler.

Reply:
That was the main reason I did not use that. @stephen-hawley  I did see you using that code, I though about it but considered th emem usage and the fact that it is an ActionDispatcher.

Reply:
How big is an empty `EventHandler` instance compared to the extra code for `null` checks?